databrowser/src/pages/DatasetRawPage.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <ContentArea>
+    <div>
+      Dataset Detail Page (type = {{ $route.params.datasetType }}, ID =
+      {{ $route.params.datasetId }})
+    </div>
+    <div>
+      <DataTable>
```

Comment:
Also here I would be more general: 
```suggestion
      <TableCustom>
```

I did the same in the button. As we cannot have `Table` alone due to Vue limitations. If you can think of a better name great, but DataTable is too specific IMO.